src/NumericalAlgorithms/Spectral/Projection.hpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Matrix.hpp"  // IWYU pragma: keep
+
+namespace Spectral {
+
+/// The portion of an element covered by a mortar.
+enum class MortarSize { Full, UpperHalf, LowerHalf };
+
+/// The projection matrix from a mortar with `num_points_source` LGL
+/// points to an element with `num_points_dest` LGL points.
```

Comment:
Could you add a `\details` section and perhaps also a `\snippet` with a bit more information on this and the context where it would be used? This would help me understand it better.

Reply:
@nilsleiffischer Is the new version better?

Reply:
yeah thanks 👍